extra/mariabackup/xtrabackup.cc

```diff
@@ -1549,6 +1550,12 @@ struct my_option xb_server_options[] =
    (G_PTR*)&srv_undo_tablespaces, (G_PTR*)&srv_undo_tablespaces,
    0, GET_ULONG, REQUIRED_ARG, 0, 0, 126, 0, 1, 0},
 
+  {"innodb_force_recovery", OPT_INNODB_FORCE_RECOVERY,
+   "(for --prepare): Crash recovery mode (ignores page corruption; for"
+   " emergencies only).",
+   (G_PTR*)&srv_force_recovery, (G_PTR*)&srv_force_recovery,
+   0, GET_ULONG, OPT_ARG, 0, 0, SRV_FORCE_IGNORE_CORRUPT, 0, 0, 0},
```

Comment:
We could add `compile_time_assert(SRV_FORCE_IGNORE_CORRUPT == 1)` somewhere.

Reply:
I think we can add that in `innodb_init_param()`, right before the assignment.